core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/start/ClusterStartActions.java

```diff
@@ -45,11 +47,21 @@ public class ClusterStartActions {
     @Inject
     private InstanceMetaDataService instanceMetaDataService;
 
-    @Bean(name = "CLUSTER_START_UPDATE_PILLAR_CONFIG_STATE")
-    public Action<?, ?> startingClusterPillarConfigUpdate() {
+    @Bean(name = "CLUSTER_DB_CERT_ROTATION_STATE")
+    public Action<?, ?> clusterDbCertRotation() {
         return new AbstractClusterAction<>(StackEvent.class) {
             @Override
             protected void doExecute(ClusterViewContext context, StackEvent payload, Map<Object, Object> variables) {
+                sendEvent(context, new ClusterDbCertRotationRequest(context.getStackId()));
+            }
+        };
+    }
+
+    @Bean(name = "CLUSTER_START_UPDATE_PILLAR_CONFIG_STATE")
+    public Action<?, ?> startingClusterPillarConfigUpdate() {
+        return new AbstractClusterAction<>(ClusterDbCertRotationResult.class) {
```

Comment:
the Payload changed, it seems an incompatible change. if we restore the state, it will fail, won't it?

Reply:
during the restart, the trigger event will go inside the CLUSTER_DB_CERT_ROTATION_STATE instead of CLUSTER_START_UPDATE_PILLAR_CONFIG_STATE. so it will work properly. 
Yes, I know It cannot be tested or is very hard to test, but the logic should work and we also add steps into the flows.